src/zino/trapd.py

```diff
@@ -97,6 +105,8 @@ def __init__(
         self._communities = set()
         self._observers: dict[TrapType, List[TrapObserver]] = {}
 
+        self.observe(ignore_trap, *IGNORED_TRAPS)
```

Comment:
Oh, this is smooth, I like it!

Reply:
What is the benefit of doing this instead of adding a check to the `dispatch_traps` function? something like

```
if (trap.mib, trap.name) in IGNORED_TRAPS:
    return
```

Reply:
Separation of concerns, really.  If the ignore procedure needs to change somehow (like adding logging or "whatever"), only the `ignore_trap` function needs to change, not the TrapReceiver.

Ideally, the ignore-observer would be registered with the trapreceiver externally, but I couldn't figure out where it would fit in without putting a lot of stuff into `zino.zino`.